src/components/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+export function patchData(article_id, body) {
+  const votes = { inc_votes: body };
+  return axios
+    .patch(
+      `https://nc-news-backend-yyld.onrender.com/api/articles/${article_id}`,
+      votes
+    )
+    .then((response) => {
+      return response;
+    })
+    .catch((err) => console.log(err));
```

Comment:
if we handle the catch here we wont be able to undo the optimistic render in the component if the request fails